src/main/java/com/acme/edu/logger/controller/MessageTypeName.java

```diff
@@ -1,4 +1,4 @@
-package com.acme.edu;
+package com.acme.edu.logger.controller;
 
 public enum MessageTypeName {
     INT("primitive"), BYTE("primitive"), CHAR("char"),
```

Comment:
While adding new type I have to make changes in existing code in some places ((( Maybe refactor to polymorphic design, TDA?